console-subscriber/src/builder.rs

```diff
@@ -83,7 +83,12 @@ impl Builder {
     /// A shorter duration will allow clients to update more frequently, but may
     /// result in the program spending more time preparing task data updates.
     ///
-    /// By default, this is [`ConsoleLayer::DEFAULT_PUBLISH_INTERVAL`].
+    /// By default, this is [`ConsoleLayer::DEFAULT_PUBLISH_INTERVAL`], though
+    /// methods like [`init`][`crate::init`] and [`spawn`][`crate::spawn`] will
+    /// take the value from the `TOKIO_CONSOLE_PUBLISH_INTERVAL` [environment
+    /// variable] before falling back on that default.
```

Comment:
how about breaking this into multiple sentences?
```suggestion
    /// By default, this is [`ConsoleLayer::DEFAULT_PUBLISH_INTERVAL`].
    /// Methods like [`init`][`crate::init`] and [`spawn`][`crate::spawn`] will
    /// take the value from the `TOKIO_CONSOLE_PUBLISH_INTERVAL` [environment
    /// variable] before falling back on that default.
```